web/cobrands/fixmystreet/fixmystreet.js

```diff
@@ -1975,3 +1975,19 @@ function setup_popstate() {
         });
     }, 0);
 }
+
+$( document ).ready(function() {
+    // add ID to each one of the multi-select-button. This will allow us to 
+    // add personalised aria-label attributes.
+    var idCnt = 1;
+    setTimeout(function() {
+        $('.multi-select-button').each(function () {
+            $(this).attr('id', function (index) {
+                return "multiselect" + idCnt;
+            });
+            idCnt++;
+        });
+        $('#multiselect1').attr('aria-label', 'Select report status');
+        $('#multiselect2').attr('aria-label', 'Select a category to display');
```

Comment:
While testing the current site I noticed that when using VoiceOver those two items weren't really clear. Regarding the elements with role="menu", I really don't think they need extra aria-labels, due to the fact that when using the screen-reader with the solution proposed in this PR, you get the following sequence using VoiceOver:

'Select report status' -> 'menu (14 items)' -> Then it starts listing all the options available while you are using the arrows(up and down) to navigate through the options.

Finally, My reason to swap the <label>Show</label> was because I don't think we need it anymore(for screen-reader), it feels like it just adds more noise, without an actual benefit.

Thanks for the feedback and let me know what you think

Reply:
Thanks for the explanation :) Not sure what you mean by "swap the Show", sorry - if you mean the HTML, then ignoring screenreaders I would like it to be semantic before any JavaScript runs (this is a WCAG requirement anyway), so I'd like the original HTML to stay the same.
If you're going to have new strings here, they'll need to be translatable with translation_strings.html please, thanks!

Reply:
My bad, I forgot to proof read the last bit, I meant the <label>show</label> element I swapped for a <span> instead.
Got it I'll keep the HTML intact, and I'll change the elements with JavaScript instead.

Reply:
> If you're going to have new strings here, they'll need to be translatable with translation_strings.html please, thanks!

In this case are talking about the strings like "Select report status" or "Select a category to display"?

Thanks again for all the help =)

Reply:
Yes, sorry, thought you'd come across this before, my bad. You'll need to add the strings to `templates/web/base/js/translation_strings.html` with IDs - see all the examples in there for the format, then refer to them in your JS as `translation_strings.YOURID`. That way, they can be translated into other languages. Hope that makes sense.

Reply:
Thanks for the explanation! =)
I just made a commit that includes the translation_strings for both of the select buttons. Now both of them have aria-labels that are pulled from the `templates/web/base/js/translation_strings.html` file.
Again the orphaned <label> elements have been replaces by <span> elements so the screen reader shouldn't have to deal with unnecessary elements anymore, instead the more clearer aria-label should provide enough info the the users to understand their purposes.

Let me know if there is any additional feedback